samples/convert-upload-images-powervs/README.md

```diff
@@ -24,13 +24,13 @@ Usage:
   ./convert-upload-images-powervs [ --rhel-url <url> | --rhcos-url <url> ] --service-name  <service name> --region <bucket region> --cos-bucket <bucket name> --cos-resource-group <resource group> [ --cos-instance-name <cos instance name> ]
 
 Args:
-      --service-name string         A service name in cloud.ibm.com you have access(Mandatory)
+      --service-name string         A list of PowerVS Compute Services with comma separated(Mandatory)
```

Comment:
this can be just `A list of PowerVS service instances with comma-separated(Mandatory)`